### PR TITLE
PICKS-236: Split score calculation script

### PR DESCRIPTION
**Pull request by mightys**
PICKS-236: Split and refactor score calculation and season create/update

### CODE REVIEW

File: server/boot/06-create-season-and-week.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          console.log('process group', group);
+          var newScores = [];
+          Season.upsertWithWhere({season: nfl.currentSeason, group: group.id},
+            {season: nfl.currentSeason, group: group.id})
+          .then(function(createdSeason) {
+            console.log('season upsert ', createdSeason);
+            group.currentSeason = createdSeason.id;
+            group.save();
+            // Create new week for season
+            Week.upsertWithWhere(
```
Comment (the-freshlord) [inline server/boot/06-create-season-and-week.js:34]: Shouldn't you be returning this promise?

**Review by the-freshlord — CHANGES_REQUESTED** [review]

File: server/boot/06-create-season-and-week.js (hunk 1 above)
Comment (sundayguru) [reply server/boot/06-create-season-and-week.js:34]: No need because it being chained to the `.then`

Comment (the-freshlord) [reply server/boot/06-create-season-and-week.js:34]: ok

Comment (the-freshlord) [reply server/boot/06-create-season-and-week.js:34]: Needs to be rebased

Comment (mightys) [reply server/boot/06-create-season-and-week.js:34]: rebased

**Review by the-freshlord — APPROVED** [review]

**Pull request merged**